I-D_in_xml/draft-ietf-ccamp-optical-impairment-topology-yang-12.txt

```diff
@@ -0,0 +1,5320 @@
+
+   *  Optical transponder function including the transceiver
+   *  Splitter/selector protection function
+
+   WDM-TE-Node-1 can be a data center device or a router router device
+   that supporting 1+1 OTSi protection for its OTs while WDM-TE-Node-2
+   is a WDM-TE-node providing add-drop ports for remote OTs as depicted
+   in Figure 9.  WDM-TE-Node-1 and WDM-TE-Node-2 are interconnected via
+   two separate TE-links, each carrying a single OTSi signal.  The
+   protection configuration for the protected TTP in WDM-TE-Node-1 can
+   be described in the same way as for use case (i) using the local-
+   link-connectivity list.
+
+
+
+
+
+
+Beller, et al.          Expires 14 September 2023              [Page 36]
+
+Internet-Draft    Opt. Impairment-Aware Topo YANG Model       March 2023
+
+
+            WDM-TE-Node-1                        WDM-TE-Node-2
+    +-----------------------------+      +---------------------------+
+    |      protected              |      |         ROADM             |
+    |      remote OT      Splitter|      |    +--------------+       |
+    |   +------------+   +--------+      |AD  |              | Line  |
+    |   |         TTP|   |     ---o----->o----o------\       | LTP 1 |
+    |   |       +----|   |    /LTP|      |LTP |       \------o-------o->
+  --o-->|       | Tx o-->o---o    |      |    |              |       |
+    |   |       +----|   |    \   |      |AD  |              |       |
+  <-o---|       | Rx o   |     ---o----->o----o---\          | Line  |
+    |   |       +----|   |     LTP|      |LTP |    \         | LTP 2 |
+    |   |            |   +--------+      |    |     \        o-------o->
+    |   +------------+            |      |    |      \       |       |
+    |                             |      o----o       \      |       |
+    |                             |      |AD  |        \     | Line  |
+    |                             |      |LTPs|         \    | LTP 3 |
+    |                             |      |    |          \---o-------o->
+    |                             |      o----o              |       |
+    |                             |      |    |              |       |
+    |                             |      |    +--------------+       |
+    +-----------------------------+      +---------------------------+
+
+
+       Figure 15: OT and OTSI protection function are both in an
+     adjacent WDM-TE-node (protected remote OT, transmit direction)
+
+2.10.2.  OMS MCG protection
+
+   [Editor's note: this section is still work in progress]
+
+   OMS MCG protection is a 1+1 protection architecture where a TE-link
+   between two adjacent WDM-TE-nodes is 1+1 protected.  Such a 1+1
+   protected TE-link composed of an underlying primary and secondary TE-
+   link is shown in Figure 16 together with an unprotected TE-link
+   between the two adjacent WDM-TE-Nodes.
+
+   1+1 OMS MCG protection is a local protection scheme, which can be
```

Comment:
It seems worthwhile referencing section 2.10.2.1. here.

Reply:
Reference to sub-section added.